README.md

```diff
@@ -120,9 +137,112 @@ module.exports = function (fastify, options, next) {
 npm run example:decorator -- -l info -w
 ```
 
-### Example: Shopify HMAC Query Parameter Verification
+# Default Methods
+
+## Default extractSignature Method
+
+The `extractSignature` method returns the HMAC signature string found in a request Signature header constructed according to [IETF draft standards][1]. This signature is compared to the calculated HMAC signature from `constructSignatureString` to validate message authenticity. This method is called with one parameter, the fastify `request` object.
```

Comment:
It might be more helpful to show an example request and how the methods interact with it. (at least the bits we need (signature header etc).

We should probably actually call this method with request + options, like we do with the others. Provides more flexibility for other users.

[nit] we shouldn't need to document the use of internal methods.

[nit] Can we break this up into separate lines haha 😄 , max chars should be ~100 I think.

Reply:
Changes made. Let me know if this is better.